msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -337,6 +337,20 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to navigate to a message in individual and group chats in Teams:
```

Comment:
Do you mean - a 1:1 chat with bot/users?

I'm not sure if we use the term "individual", double check.

Reply:
can we just say to a message in chat in Teams?

Reply:
can we just say to a message in chat in Teams?

Reply:
Updated individual chat to personal chat.